lib/basic_assumption.rb

```diff
@@ -113,6 +113,6 @@ def define_writer_method(name)
   end
 end
 
-if defined?(Rails) && Rails::VERSION::MAJOR == 3
+if defined?(Rails)
```

Comment:
Just for a smaller change, what about switching to `if defined?(Rails) && Rails::VERSION::MAJOR >= 3`?